tst/inet/mqtt_client/main.c

```diff
@@ -357,6 +357,352 @@ static int test_subscribe(struct harness_t *harness_p)
     return (0);
 }
 
+static int test_incoming_publish_qos0(struct harness_t *harness_p)
+{
+    uint8_t buf[16];
+    struct message_t message;
+    struct mqtt_application_message_t foobar;
+
+    /* Prepare the server to receive the subscribe message. */
```

Comment:
Isn't is possible to remove all subscribe logic, and only keep publish logic, in these testcases?